src/Uno.UI/Controls/NativeFramePresenter.iOS.cs

```diff
@@ -363,7 +363,10 @@ private void ForceFrameStateIntoNavigationController()
 					this.Log().Debug("Resetting all ViewControllers based on Frame's state.");
 				}
 
-				NavigationController.SetViewControllers(viewControllers, animated: true);
+				// TODO: iOS 14 introduced a bug where calling this method is getting unpleasant results (https://developer.apple.com/forums/thread/656524). (https://developer.apple.com/forums/thread/656524)
```

Comment:
The comment starts with "TODO:", but it there really something to do?

Reply:
Follow up to check on iOS. That's the only thing we could DO.